spec/clients_spec.rb

```diff
@@ -78,6 +78,45 @@
       client.add_product_id(key: 'key', id: 'id')
       expect { client.clear_product_ids }.to change { client.product_ids.length }.to(0)
     end
+  end
+
+  context 'when testing #do_get' do
```

Comment:
For the context, testing is not needed because the it's testing already.
Instead, we can say something like `when sending http request through #do_get`